ha/setup/ha_setup.py

```diff
@@ -791,6 +790,69 @@ def process(self):
         """
         pass # TBD
 
+
+class PreUpgradeCmd(Cmd):
+    """
+    Pre-Upgrade Cmd
+    """
+    name = "pre_upgrade"
+
+    def __init__(self, args):
+        """
+        Init method.
+        """
+        super().__init__(args)
+
+    def process(self):
+        """
+        Process command.
+
+        NOTE: Consul is not expected to be upgraded, so no need to backup
+        consul config
+        Configuration backup is performed for every node while pacemaker setup
+        only on one node (cluster level).
+        """
+        try:
+            if os.environ['PRVSNR_MINI_LEVEL'] == 'cluster':
+                Log.info("Performing pre disruptive upgrade routines on cluster \
+                         level")
+                cluster_standby_mode()
```

Comment:
cluster level pre-upgrade hook for offline upgrade is called at the time when cluster is already in standby mode. Is it safe to standby again here?

Reply:
Technically it is safe, but I can just remove this invocation and rely that Provisioner will put cluster to standby.